android/tutorials/tutorial3/README.md

```diff
@@ -275,12 +275,29 @@ Building the project at this point, Dagger should give you:
 
 ## Bubbling up dependencies
 
-This will be super easy, if you got this far. The only difference is that if we cannot provide a dependency directly, we also add it to the `Dependency` interface of the parent (and then to its parent and so on until we can actually provide id).
+This will be super easy, if you got this far. The only difference is that if we cannot provide a dependency directly, we also add it to the `Dependency` interface of parent and to the 'Dependency' interfaces of all parent ribs until this rib can actually provide id.
 
 Really, just a one-liner.
 
 Try it:
 
+```kotlin
+interface HelloWorld : Rib {
```

Comment:
This is still out of context regarding the text before and I wouldn't include it here. Enough to have it in solutions.

Reply:
done